core/regression_test/regression_test.jl

```diff
@@ -0,0 +1,300 @@
+@testitem "regression_ode_solvers_trivial" begin
+    using SciMLBase: successful_retcode
+    using Tables: Tables
+    using Tables.DataAPI: nrow
+    using Dates: DateTime
+    import Arrow
+    using Ribasim: get_tstops, tsaves
+
+    toml_path = normpath(@__DIR__, "../../generated_testmodels/trivial/ribasim.toml")
+    @test ispath(toml_path)
+
+    # There is no control. That means we don't write the control.arrow,
+    # and we remove it if it exists.
+    control_path = normpath(dirname(toml_path), "results/control.arrow")
+    mkpath(dirname(control_path))
+    touch(control_path)
+    @test ispath(control_path)
+    config = Ribasim.Config(toml_path)
```

Comment:
These are more like unit tests, that don't need to be copied over from `run_models_test.jl`.

Reply:
The content is copied from `run_model.tests`, but it tests the model under 7 different solvers and 4 different settings. This type of tests can be run in the regression test set instead of unit test set